curations/pypi/pypi/-/pywin32.yaml

```diff
@@ -39,3 +39,6 @@ revisions:
   '303.1':
     licensed:
       declared: PSF-2.0
+  '304':
+    licensed:
+      declared: BSD-3-Clause
```

Comment:
This is incorrect. It's `PSF-2.0` still.

Reply:
I agree with PSF-2.0